### PR TITLE
Add moment.js facade

### PR DESCRIPTION
**Pull request by chuwy**

### CODE REVIEW

File: _data/library/jsfacades.yml [hunk 1]
```diff
@@ -64,6 +64,10 @@
       url: https://github.com/coreyauger/scala-js-youtube
       desc: Type safe and Scala friendly library for the embedded YouTube player API
       dep: ''
+    - name: scala-js-momentjs
+      url: https://github.com/widok/scala-js-momentjs
+      desc: [Moment.js](http://momentjs.com/) façade
```
Comment (sjrd) [inline _data/library/jsfacades.yml]: We typically use the `facade` variant of the word (without the 'ç' cedilla).

**Comment by sjrd** [comment]
Could you replace the ç by a c and *amend* your commit, please?

**Comment by chuwy** [comment]
Done!

**Comment by sjrd** [comment]
LGTM, thanks!

**Comment by chuwy** [comment]
Seems I also need to take dep value in quotes. Does Travis complains may caused by this?

File: _data/library/jsfacades.yml [hunk 2]
```diff
@@ -64,6 +64,10 @@
       url: https://github.com/coreyauger/scala-js-youtube
       desc: Type safe and Scala friendly library for the embedded YouTube player API
       dep: ''
+    - name: scala-js-momentjs
+      url: https://github.com/widok/scala-js-momentjs
+      desc: [Moment.js](http://momentjs.com/) facade
+      dep: libraryDependencies += "io.github.widok" % "scala-js-momentjs_sjs0.6_2.11" % "0.1.4"
```
Comment (sjrd) [inline _data/library/jsfacades.yml]: Oh, I think Travis doesn't like this line. Not to mention that it doesn't have the best format. It should be:

    dep: '"io.github.widok" % "scala-js-momentjs_sjs0.6_2.11" % "0.1.4"'

with enclosing `'` and no `libraryDependencies +=`

Comment (chuwy) [inline _data/library/jsfacades.yml]: Sorry. Fixed.

Comment (sjrd) [inline _data/library/jsfacades.yml]: And the `%%%` formatting:

    dep: '"io.github.widok" %%% "scala-js-momentjs" % "0.1.4"'

**Comment by sjrd** [comment]
Hum ... I can actually reproduce the build failure locally, but I have *no idea* why it doesn't parse :s

File: _data/library/jsfacades.yml [hunk 3]
```diff
@@ -64,6 +64,10 @@
       url: https://github.com/coreyauger/scala-js-youtube
       desc: Type safe and Scala friendly library for the embedded YouTube player API
       dep: ''
+    - name: scala-js-momentjs
+      url: https://github.com/widok/scala-js-momentjs
+      desc: [Moment.js](http://momentjs.com/) facade
```
Comment (sjrd) [inline _data/library/jsfacades.yml]: Found it! Add `'` around this line too. Apparently it doesn't like that the value *starts* with a `[`

**Comment by chuwy** [comment]
Omg. Fixed. So sorry :)

File: _data/library/jsfacades.yml (hunk 3 above)
Comment (ochrons) [inline _data/library/jsfacades.yml]: I guess YAML considers something starting with a `[` as an array.

**Comment by sjrd** [comment]
LGTM. No problem! I didn't know that either ;)

**Pull request merged**